### ISSUE TITLE
The swap-in screen could use a link to block explorer

### ISSUE THREAD

**Issue by jesterhodl** [ACINQ/phoenix#367]
Hi, when you accept onchain address to swap-in and you're waiting for the funds to arrive, it would be good to have a link handy on the screen to look it up on mempool.space.

Attaching the relevant screen

![IMG_4048](https://github.com/ACINQ/phoenix/assets/103882255/71f56ad1-e71b-40d7-99c2-ccd5948e94c4)

**Comment by dpad85** [comment]
Good idea, this flow will change quite a bit with splicing which makes it much easier to track what's happening with swaps, but this can still be useful.

**Comment by dpad85** [comment]
Since Phoenix v2, the pending on-chain transactions to the swap-in wallet can be viewed in Settings > Wallet info > Swap-in wallet, and can be clicked to open an explorer.
